src/components/RegisterTab/RegisterTab.tsx

```diff
@@ -95,9 +99,15 @@ export class RegisterTab extends Component<RegisterTabProps, {personalEmailTouch
           Please type in format of XXX-XXX-XXXX
         </Form.Control.Feedback>
       </Form.Group>
-
       <Form.Group controlId='formBasicPassword'>
-        <Form.Label column={true}>Password</Form.Label>
+        <Form.Label column={true}>New Password</Form.Label>
+        <p style={pStyle}>
```

Comment:
I would put this below the password box.  Use bootstrap's example to add text below the input: https://react-bootstrap.github.io/components/forms/#forms

Why did you rename the password to "New Password"? This is the registration, not the change password page. You will need to do this on the change password page as well.

Change the text to:
```
We require a strong password for using our system. To increase the password strength, add numbers, special character, and increase password length.
```

Reply:
I was copying and pasting from both the registration page and the reset password page and I think I missed that when I pasted.